hooli-data-eng/hooli_data_eng/components/scheduled_job_component.py

```diff
@@ -12,22 +13,13 @@ class ScheduledJobComponent(Component, Resolvable):
 
     # added fields here will define yaml schema via Model
     cron_schedule: str
-    dagster_selection: str
+    asset_selection: str
```

Comment:
oh this is 💯  better -- when I first wrote this it was "dbt_selection" but really it's just "asset_selection".